### ISSUE TITLE
Error in next/image with high aspect ratio images

### ISSUE THREAD

**Issue by jlabedo** [vercel/next.js#40602]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

    Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:29 PDT 2022; root:xnu-8020.121.3~4/RELEASE_ARM64_T8101
    Binaries:
      Node: 18.3.0
      npm: 8.11.0
      Yarn: 3.1.1
      pnpm: 7.5.0
    Relevant packages:
      next: 12.3.0
      eslint-config-next: 12.3.0
      react: 17.0.2
      react-dom: 17.0.2

### Describe the Bug

With high aspect ratio images and small height, we can get an error during build while optimising the image:

```
Error: Expected positive integer for height but received 0 of type number
    at Object.invalidParameterError (/app/node_modules/sharp/lib/is.js:124:10)
    at Sharp.resize (/app/node_modules/sharp/lib/resize.js:251:16)
    at Object.resizeImage (/app/node_modules/next/dist/server/image-optimizer.js:623:21)
    at /app/node_modules/next/dist/build/webpack/loaders/next-image-loader.js:63:98
    at Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:79:26)
    at /app/node_modules/next/dist/build/webpack/loaders/next-image-loader.js:63:60
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at runNextTicks (node:internal/process/task_queues:65:3)
    at processImmediate (node:internal/timers:437:9)
```

In my case it is a 184x8 px image.

### Expected Behavior

No error during build.

This is because the shrinked image height for bluring is rounded to 0:
https://github.com/vercel/next.js/blob/755661144979db84e44caa5bb66b62e350f8a107/packages/next/build/webpack/loaders/next-image-loader.js#L38-L40

Could be fixed with:
```js
blurHeight = Math.max(1, Math.round(
     (imageSize.height / imageSize.width) * BLUR_IMG_SIZE
))
```

Same for the width on next lines.

### Link to reproduction

irrelevant

### To Reproduce

import a static image with high aspect ratio and low height. The build will fail

**Comment by SukkaW** [comment]
Duplicated with #40562 and should be fixed by #40563.

**Comment by balazsorban44** [comment]
Closing, as pointed out, this is a duplicate of #40562
